packages/cli/src/commands/remove.ts

```diff
@@ -110,7 +116,7 @@ export default async function main(client) {
     return 1;
   }
 
-  let contextName = null;
+  let contextName: string | null = null;
```

Comment:
```suggestion
  let contextName;
```

I think you can let TS infer the type here and the early return will let it work. Then you can remove the `!` on 147

Reply:
No working this solution
Error:
> Variable 'contextName' implicitly has type 'any' in some locations where its type cannot be determined.ts(7034)